.travis.d/definition_check.py

```diff
@@ -0,0 +1,67 @@
+import os
+import sys
+import re
+import fnmatch
+
+CURR_DIR = os.path.dirname(os.path.realpath(__file__))
+DEF_FILE_PATH = os.path.join(CURR_DIR, '..', 'docs', 'definitions.dox')
+INCLUDE_DIR = os.path.join(CURR_DIR, '..', 'include')
+
+DEFINE_RULE = re.compile(r' \*     \\defgroup ([a-zA-Z]+) [a-zA-Z ]+')
+USER_GROUP_RULE = re.compile(r' \* \\(user|kernel)group\{([a-zA-Z]+)\}')
```

Comment:
1. we should detect wrong indentation.
2. but your opinion is sometime we should pass that.

i'm not agree these conflict rules.
reviewer can suggest to contributors.